pallets/creditcoin/src/lib.rs

```diff
@@ -1093,6 +1175,41 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[transactional]
+		#[pallet::weight(<T as Config>::WeightInfo::request_collect_coins())]
+		pub fn request_collect_coins(
+			origin: OriginFor<T>,
+			evm_address: ExternalAddress,
+			tx_id: ExternalTxId,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
```

Comment:
Q: do we also need `ensure!(Authorities::<T>::contains_key(&who), Error::<T>::InsufficientAuthority);` here like in `persist_collect_coins` and `fail_collect_coins` ?